index.php

```diff
@@ -2,13 +2,17 @@
 
 require_once 'functions.php';
 
+$item = (isset($_GET['item'])) ? $_GET['item'] : NULL;
```

Comment:
If these are null what happens?

Reply:
nothing happens

Reply:
Ok cool, so there should be some sort of error/feedback to the user in addition to not inserting into the database